### PR TITLE
Add LW-12 documentation

### PR DESCRIPTION
**Pull request by fabaff**
**Description:**
Add LW-12 documentation

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#13307

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Comment by jaypikay** [comment]
I have removed all but host, port and name as valid options in the entitiy module.

File: source/_components/light.lw12wifi.markdown [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+description: "Instructions on how to setup Lagute LW-12 Wifi LED controller within Home Assistant."
+date: 2018-04-28 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Light
+ha_iot_class: "Local Polling"
+logo: lagute.png
+ha_release: 0.69
```
Comment (MartinHjelmare) [inline source/_components/light.lw12wifi.markdown]: 0.71

File: source/_components/light.lw12wifi.markdown [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+---
+
+The `lw12wifi` light platofrm supports Lagute LW-12 Wifi LED controller.
```
Comment (MartinHjelmare) [inline source/_components/light.lw12wifi.markdown]: ... light platform supports...

File: source/_components/light.lw12wifi.markdown [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+## {% linkable_title Configuration %}
+
+To enable these lights, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+light:
+  - platform: decora
```
Comment (MartinHjelmare) [inline source/_components/light.lw12wifi.markdown]: lw12wifi

File: source/_components/light.lw12wifi.markdown [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+  description: Some firmware versions of the LW-12 controller listen on different ports.
+  required: false
+  type: int
+  default: 5000
+name:
+  description: Name to use in the frontend.
+  required: false
+  type: string
+  default: LW-12 FC
+transistion:
```
Comment (MartinHjelmare) [inline source/_components/light.lw12wifi.markdown]: Remove the following config options. Those weren't implemented.

**Review by balloob — APPROVED** [review]

**Pull request merged**